services/galley/test/integration/API/SQS.hs

```diff
@@ -33,22 +35,28 @@ import qualified Network.AWS.SQS as SQS
 import qualified OpenSSL.X509.SystemStore as Ssl
 import qualified System.Logger as L
 
-ensureQueueEmpty :: MonadIO m => Maybe Aws.Env -> m ()
-ensureQueueEmpty (Just env) = liftIO $ Aws.execute env purgeQueue
-ensureQueueEmpty Nothing    = return ()
+ensureQueueEmpty :: TestM ()
+ensureQueueEmpty  = view TS.awsEnv >>= ensureQueueEmptyIO
 
-assertQueue :: MonadIO m => String -> Maybe Aws.Env -> (String -> Maybe E.TeamEvent -> IO ()) -> m ()
-assertQueue label (Just env) check = liftIO $ Aws.execute env $ fetchMessage label check
-assertQueue _     Nothing    _     = return ()
+ensureQueueEmptyIO :: MonadIO m => Maybe Aws.Env -> m ()
```

Comment:
This is the only one which is used outside of TestM in main; this seemed the easiest approach.